README.md

```diff
@@ -1199,6 +1199,13 @@ class TritonPythonModel:
     # tensor.
     input0 = pb_utils.Tensor.from_dlpack("INPUT0", to_dlpack(pytorch_tensor))
 ```
+Starting from 23.04 release, Python backend allows external tensors being passed 
+directly to `pb_utils.Tensor.from_dlpack()` without an explicit DLPack encoding. 
+For instance:
```

Comment:
Starting from 23.04 release, Python backend allows tensors that implement `__dlpack__` and `__dlpack_device__` interface to be converted to Python backend tensors.